### ISSUE TITLE
Eldewrito 0.6.1 with DXVK causes GPU reset (amdgpu_job_timedout)

### ISSUE THREAD

**Issue by DomiStyle** [doitsujin/dxvk#3184]
When starting Eldewrito 0.6.1 (aka Halo Online) with DXVK the game will freeze at the start screen for a few seconds before the X11 session crashes.

Sometimes (especially on Wayland) the game will start but performance will be abysmal and looking at the syslog reveals a bunch of soft resets every 20-30s.

The game launches fine with Gallium Nine and WineD3D, albeit with some graphical and performance issues.

I can provide download links for the game files since it's a hassle to patch them on Linux.

### Software information
Eldewrito 0.6.1, default settings

### System information
- GPU: AMD 6900 XT
- Driver: Mesa 23.0.0-devel (LLVM 15.0.6)
- Wine version: lutris-GE-Proton7-35-x86_64 & lutris-6.2-2-x86_64
- DXVK version: 2.0

### Apitrace file(s)

Apitrace with Gallium Nine:
https://mega.nz/file/vFoVAaQK#bx_dbeqZw8EbZLbuD065kWmi2lboYmnaQFxIZyByr58

### Log files
I tried to get DXVK debug logs and RADV trace files but none are created even though `DXVK_LOG_PATH` and `RADV_TRACE_FILE` are set. Just tell me if they (or something else) are needed and I will take another look.

syslog for the amdgpu timeout:
[halo_crash.txt](https://github.com/doitsujin/dxvk/files/10393153/halo_crash.txt)

**Comment by mbriar** [comment]
Can you maybe try this with amdvlk as well? GPU hangs are often vulkan driver bugs.

**Comment by DomiStyle** [comment]
Same issue here with AMDVLK.

```
Jän 12 00:43:06 domi-pc net.lutris.Lutris.desktop[3841]: debug: Compiling graphics pipeline...
Jän 12 00:43:06 domi-pc net.lutris.Lutris.desktop[3841]: debug:   vs  : VS_1a586b378d63907f6c6cf4c8913ca281a5d37758
Jän 12 00:43:06 domi-pc net.lutris.Lutris.desktop[3841]: debug:   fs  : FS_b4806b860cdbaf2975cf4f8b12b95d0d000f379b
Jän 12 00:43:06 domi-pc net.lutris.Lutris.desktop[3841]: debug:   attr 0 : location 0, binding 0, format VK_FORMAT_R32G32_SFLOAT, offset 0
Jän 12 00:43:06 domi-pc net.lutris.Lutris.desktop[3841]: debug:   attr 1 : location 1, binding 0, format VK_FORMAT_R32G32_SFLOAT, offset 8
Jän 12 00:43:06 domi-pc net.lutris.Lutris.desktop[3841]: debug:   binding 0 : binding 0, stride 20, rate 0, divisor 0
Jän 12 00:43:12 domi-pc NetworkManager[1313]: <info>  [1673480592.9312] manager: NetworkManager state is now CONNECTED_SITE
Jän 12 00:43:12 domi-pc systemd[1]: Starting NetworkManager-dispatcher.service - Network Manager Script Dispatcher Service...
Jän 12 00:43:13 domi-pc NetworkManager[1313]: <info>  [1673480593.2790] manager: NetworkManager state is now CONNECTED_GLOBAL
Jän 12 00:43:16 domi-pc kernel: [drm:amdgpu_job_timedout [amdgpu]] *ERROR* ring gfx_0.0.0 timeout, signaled seq=310421, emitted seq=310423
Jän 12 00:43:16 domi-pc kernel: [drm:amdgpu_job_timedout [amdgpu]] *ERROR* Process information: process eldorado.exe pid 10431 thread eldorado.exe pid 10450
Jän 12 00:43:16 domi-pc kernel: amdgpu 0000:0c:00.0: amdgpu: GPU reset begin!
Jän 12 00:43:17 domi-pc kernel: amdgpu 0000:0c:00.0: [drm:amdgpu_ring_test_helper [amdgpu]] *ERROR* ring kiq_2.1.0 test failed (-110)
Jän 12 00:43:17 domi-pc kernel: [drm:gfx_v10_0_hw_fini [amdgpu]] *ERROR* KGQ disable failed
Jän 12 00:43:17 domi-pc kernel: [drm:gfx_v10_0_hw_fini [amdgpu]] *ERROR* failed to halt cp gfx
Jän 12 00:43:17 domi-pc kernel: [drm] free PSP TMR buffer
Jän 12 00:43:17 domi-pc kernel: amdgpu 0000:0c:00.0: amdgpu: MODE1 reset
Jän 12 00:43:17 domi-pc kernel: amdgpu 0000:0c:00.0: amdgpu: GPU mode1 reset
Jän 12 00:43:17 domi-pc kernel: amdgpu 0000:0c:00.0: amdgpu: GPU smu mode1 reset
Jän 12 00:43:17 domi-pc kernel: amdgpu 0000:0c:00.0: amdgpu: GPU reset succeeded, trying to resume
```

**Comment by DomiStyle** [comment]
Done, opened an issue here: https://gitlab.freedesktop.org/mesa/mesa/-/issues/8067

**Comment by DomiStyle** [comment]
Issue appears to be fixed with Mesa 24.0.3 and later.
